tests/suites/host_test.function

```diff
@@ -498,7 +498,8 @@ int execute_tests( int argc , const char ** argv )
 
             if( ( ret = get_line( file, buf, sizeof(buf) ) ) != 0 )
                 break;
-            mbedtls_fprintf( stdout, "%s%.66s", test_info.failed ? "\n" : "", buf );
+            mbedtls_fprintf( stdout, "%s%.66s",
+                    test_info.result == TEST_RESULT_FAILED ? "\n" : "", buf );
```

Comment:
Should this be `test_info.result != TEST_RESULT_SUCCESS`?

Reply:
As far as I understand, we want a line break on a failing test so that on the next line we can print the detailed failure information. While, for a successful or a skipped test, we want to print `PASS` or `----` at the end of the line.